avocado/core/multiplexer.py

```diff
@@ -377,28 +377,67 @@ def iteritems(self):
                 yield (leaf.environment_origin[key].path, key, value)
 
 
+def _report_already_parsed(self, *args, **kwargs):
+    """
+    Raises exception describing that `self.data` alteration is restricted
+    """
+    raise RuntimeError("Altering of `self.data` is restricted. %s %s"
```

Comment:
IMHO, it doesn't make sense to mention `self.data` explicity, since it's not something we expect to support in the public API, right?

Also, `args` and `kwargs` are just listed/printed, without saying, for example, that they were the values inteded to me injected/merged.